src/json/members/yuclin.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Yu-Hsin Lin",
+  "pennkey": "yuclin",
+  "major": "Integrated Product Design",
+  "school": "Penn Engineering",
+  "bio": "Exploring Philly and bubble teas!",
+  "hometown": "Taipei, Taiwan",
+  "team": "Design",
```

```suggestion
  "team": "Penn Clubs",
```

There currently isn't a Design team on the website, so we're going to put you under the Penn Clubs section for now given that they'll probably be one of the first teams to work with the new design system.